### PR TITLE
Pay 43818 invoice field changes

### PR DESCRIPTION
**Pull request by TadasAmbrazaitis**
Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
If the fields are not filled, they are not displayed in PDF.
Can be negative: TotalVATAmount, TotalPriceIncludingVat, PricePerUnit, PriceDiscount.
More than 2 decimals allowed for: Quantity, PricePerUnit.

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Subscriptions API Release Notes
+<div class='post-date'>04 November 2020 - Invoice fields</div> 
+Merchants can add [invoice](invoice) for susbscription payments without user's information.
+Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
+If the fields are not filled, they are not displayed in PDF.
+Can be negative: TotalVatAmount, TotalPriceIncludingVat, PricePerUnit, PriceDiscount.
```
Comment (edvinasmuncelis) [inline docs/release_notes.md:6]: these fields have snake case in the contract, don't they? If so, don't use different casing here.

File: docs/release_notes.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+10 decimals allowed for: Quantity, PricePerUnit.
```
Comment (edvinasmuncelis) [inline docs/release_notes.md:7]: "10 places after the separator" ? You don't allow 10 different numbers here, I guess

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,4 +1,11 @@
 # Subscriptions API Release Notes
+<div class='post-date'>04 November 2020 - Invoice fields</div> 
+Merchants can add [invoice](invoice) for susbscription payments without user's information.
+Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
```
Comment (edvinasmuncelis) [inline docs/release_notes.md:4]: *"Fields that are now optional"

Comment (edvinasmuncelis) [reply docs/release_notes.md:4]: By the way, isn't the term "property" more appropriate in this context instead of "field" ?

**Review by edvinasmuncelis — CHANGES_REQUESTED** [review]

File: docs/release_notes.md (hunk 2 above)
Comment (TadasAmbrazaitis) [reply docs/release_notes.md:7]: Decimal also refers specifically to the digits after the decimal separator, such as in "3.14 is the approximation of π to two decimals".
I think all of the error messages in RecurringPayments use this kind of naming "no more than 10 decimals" and etc. Is it too ambiguous?

File: docs/release_notes.md (hunk 3 above)
Comment (TadasAmbrazaitis) [reply docs/release_notes.md:4]: From my understanding - merchants fill the fields and then properties get filled from the fields. Which naming should I choose?

File: docs/release_notes.md (hunk 2 above)
Comment (TadasAmbrazaitis) [reply docs/release_notes.md]: In the new commit I made it as easily understandable as possible.

File: docs/release_notes.md (hunk 3 above)
Comment (TadasAmbrazaitis) [reply docs/release_notes.md]: In the end I went with properties.

Comment (edvinasmuncelis) [reply docs/release_notes.md]: @TadasAmbrazaitis merchants are sending a JSON with properties.

**Comment by edvinasmuncelis** [comment]
Approved.

**Pull request merged**